src/components/PostcodeLookup/PostcodeLookup.js

```diff
@@ -334,6 +337,7 @@ class PostcodeLookup extends Component {
    * @return {*}
    */
   render() {
+    // console.log(this.state.isHidden);
     const postCodeField = {
```

Comment:
just a reminder that there's a comment out line here.

Reply:
oops that shouldnt be there